app/views/katello/api/v2/systems/base.json.rabl

```diff
@@ -31,5 +31,5 @@ attributes :checkin_time, :created
 attributes :installedProducts
 
 node :errata_counts do |system|
-  partial('katello/api/v2/errata/counts', :object => Katello::RelationPresenter.new(system.installable_errata))
-end
+  partial('katello/api/v2/errata/counts', :object => Katello::RelationPresenter.new(system.foreman_host.content_facet.installable_errata))
+end
```

Comment:
can you update https://github.com/Katello/katello/blob/master/app/views/katello/api/v2/content_facet/base.json.rabl#L8 too?

might need to rebase against master to get that